src/sasctl/_services/score_definitions.py

```diff
@@ -83,15 +83,20 @@ def create_score_definition(
         else:
             object_descriptor_type = "sas.models.model.ds2"
 
-        model = cls._model_repository.get_model(model)
+        if cls._model_repository.is_uuid(model):
+            model_id = model
+        elif isinstance(model, dict) and "id" in model:
+            model_id = model["id"]
+        else:
+            model = cls._model_repository.get_model(model)
+            model_id = model["id"]
```

Comment:
Will `model["id"]` throw an error if the `get_model(model)` returns None? Should we move the HTTPError to the else statement?